.circleci/config.yml

```diff
@@ -28,12 +32,6 @@ aliases:
     environment:
       <<: *e2e-executor-env
 
-  restore_cache: &restore_cache
```

Comment:
these two were CircleCI "aliases" which can't accept parameters (node version), so I changed them to CircleCI "commands"